src/openloco/sv5.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
+
+#include "compat.h"
+#include <cstdint>
+namespace openloco::sv5
+{
+#pragma pack(push, 1)
+    struct ScenarioConfiguration
+    {
+        uint8_t editor_step; // 0x00
+        uint8_t pad_1;
+        uint16_t scenarioStartYear; // 0x02
+        uint8_t pad_4[2];
+        uint16_t scenarioFlags; // 0x06
+        uint8_t var_9C871C;     // 0x08
+        uint8_t pad_9[1];
+        uint8_t landObjectDiversity[32]; // 0x0A
+        uint8_t pad_2A[336];
```

Comment:
`0x009C873E` is the start of a buffer for the scenario name (length 64, incl. `\0`);
`0x009C877E` is the start of a buffer for the scenario description.